src/Entity/Book.php

```diff
@@ -55,10 +55,133 @@ public function __construct()
         $this->physicalBooks = new ArrayCollection();
         $this->reviews = new ArrayCollection();
         $this->borrows = new ArrayCollection();
+        $this->id = Uuid::v4();
     }
 
     public function getId(): Uuid
     {
         return $this->id;
     }
+
+    
+
+    /**
+     * Get the value of writerFirstname
+     */ 
+    public function getWriterFirstname()
+    {
+        return $this->writerFirstname;
+    }
+
+    /**
+     * Set the value of writerFirstname
+     *
+     * @return  self
```

Comment:
Useless, add a return type instead.
By the way, I would not introduce a fluent interface here. Just don't return anything.

Same for all accessor methods.